src/server/bitops_family.cc

```diff
@@ -1237,19 +1237,9 @@ void SetBit(CmdArgList args, ConnectionContext* cntx) {
 
 // ------------------------------------------------------------------------- //
 // This are the "callbacks" that we're using from above
-std::string GetString(const PrimeValue& pv, EngineShard* shard) {
+std::string GetString(const PrimeValue& pv) {
```

Comment:
Seems like this would better be in `PrimeValue` :)

Reply:
not sure I understand the comment..
do you mean in compact object?

Reply:
yes :)